src/Server/Server.fsproj

```diff
@@ -20,10 +20,7 @@
       <Paket>True</Paket>
     </Content>
 
-    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.0-beta-060000" PrivateAssets="All" />
-    <DotNetCliToolReference Include="dotnet-compile-fsc" Version="1.0.0-preview2-020000" />
-
-    <EmbeddedResource Include="**\*.resx" />
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
```

Comment:
Well, we do need to update the dotnet sdk version ;) but anyways I can do it in another PR.